cloudify/deployments.go

```diff
@@ -161,3 +163,23 @@ func (cl *Client) CreateDeployments(deploymentID string, depl DeploymentPost) (*
 
 	return &deployment, nil
 }
+
+func (cl *Client) GetDeploymentByInstance(instanceID string) (string, error) {
+	var params  = map[string]string{}
+	var deploymentID = ""
+
+	nodeInstances, err := cl.GetNodeInstances(params)
```

Comment:
use params['id'] = instanceID for limit results

https://github.com/cloudify-incubator/cloudify-rest-go-client/blob/master/cfy-go/instances.go#L123